config/config.txt

```diff
@@ -467,16 +467,16 @@ AUTOADMIN_RANK Admin
 ## These trigger for any version below (non-inclusive) the given version, so 510 triggers on 509 or lower.
 ## These messages will be followed by one stating the clients current version and the required version for clarity.
 ## If CLIENT_WARN_POPUP is uncommented a popup window with the message will be displayed instead
-#CLIENT_WARN_VERSION 513
-#CLIENT_WARN_BUILD 1421
+CLIENT_WARN_VERSION 515
+CLIENT_WARN_BUILD 1640
 #CLIENT_WARN_POPUP
-#CLIENT_WARN_MESSAGE 512 is no longer being directly supported as version 513 is set to become the new stable version soon. We've made a number of changes to take advantage of the improvements made in 513 which should make for a smoother experience. We will be removing support for 512 when this new version replaces it as stable, so it's recommended that you upgrade now. (You can update to the BETA via the website or directly in the BYOND client)
+CLIENT_WARN_MESSAGE Your current client version does not support all features used by beestation, and will have some elements such as the alt-click stat panel menu simplified. Please update to 515, as it will become mandatory in the future.
```

Comment:
```suggestion
CLIENT_WARN_MESSAGE Your current client version does not support all features used by the server, and will have some elements such as the alt-click stat panel menu simplified. Please update to 515, as it will become mandatory in the future.
```